src/core_modules/capture-core/components/WidgetProfile/hooks/getSubValueForTei.js

```diff
@@ -48,9 +48,9 @@ const getImageResourceSubvalue = async ({ attribute, minorServerVersion }: SubVa
 };
 
 const getOrganisationUnitSubvalue = async ({ attribute: { value }, querySingleResource }: SubValueFunctionParams) => {
-    const orgUnits = await getOrgUnitNames([value], querySingleResource);
+    const organisationUnit = await getOrgUnitNames([value], querySingleResource);
```

Comment:
To make the return statement below clearer I think `organisationUnit` should be renamed to `organisationUnits` or `orgUnits`. Same in getSubvalueTofTei.js.